docs/topics/Gateway.md

```diff
@@ -282,6 +283,9 @@ Once connected, the client should immediately receive an [OP 10 Hello](#DOCS_GAT
 
 Next the client is expected to send an [OP 2 Identify](#DOCS_GATEWAY/gateway-identify) _or_ [OP 6 Resume](#DOCS_GATEWAY/gateway-resume) payload. If the token passed is correct, the gateway will respond with a [Ready](#DOCS_GATEWAY/ready) event, and your client can be considered in a "connected" state. Clients are limited to 1 identify every 5 seconds, if they exceed this limit the gateway will respond with an [OP 9 Invalid Session](#DOCS_GATEWAY/gateway-invalid-session) and terminate the connection. It is important to note that although the ready event contains a large portion of the required initial state, some information (such as guilds and their members) is asynchronously sent (see [Guild Create](#DOCS_GATEWAY/guild-create) event)
 
+>warn
+> Clients are limited to 1000 `IDENTIFY` calls to the websocket in a 24-hour period. This limit is global and across all shards, but does not include `RESUME` calls. Any client that hits that limit will automatically have their bot token reset, and the operator must update their application accordingly. This is to prevent bots from getting into infinite restart loops. When the limit is reached, the owner will receive an email stating that their bot token has been reset, and all active sessions for that bot will be terminated.
```

Comment:
#215 Notes "successfully" but does not say whether or not success is contingent on receiving an acknowledgement (`READY`) or simply that the call happened. I will check and clarify if necessary